src/app/GitUI/UserControls/RevisionGrid/Columns/MessageColumnProvider.cs

```diff
@@ -329,7 +329,7 @@ void DrawSuperProjectRef(string label, ref int currentOffset, bool isSelected)
                     style.NormalFont,
                     ref currentOffset,
                     label,
-                    Color.OrangeRed,
+                    Color.OrangeRed.AdaptBackColor(),
```

Comment:
❓
```suggestion
                    headColor: Color.OrangeRed.AdaptTextColor(),
```
because of
```
Color textColor = fill ? headColor : ColorHelper.Lerp(headColor, Color.Black, 0.25F);
```

Reply:
This is hov it renders with Back:
![image](https://github.com/user-attachments/assets/ea737ffc-cbcb-4a9b-940e-39190f020254)

With Text (Fore)
![image](https://github.com/user-attachments/assets/d1cab0f2-8369-46e0-b5a6-b105f3557af9)

will change